README.md

```diff
@@ -84,6 +84,10 @@ Options:
                  negative it's considered from feed end                 [number]
   --end          feed read end, ignored in case if tail is specified, if
                  negative it's considered from feed end                 [number]
+  --start-date   feed read start by date, ignored in case if start
+                 is specified                                           [string]
+  --end-date     feed read end by date, ignored in case if end
+                 is specified                                           [string]
```

Comment:
can we add an example under `## Examples` section, this feature is really useful